Package.swift

```diff
@@ -404,13 +404,8 @@ let package = Package(
             path: "src/realm/parser",
             exclude: [
                 "CMakeLists.txt",
-                "driver.hpp",
```

Comment:
These header exclusions were a leftover workaround for an Xcode 12 bug that I forgot to remove.